diff -urp flex-2.5.34/tests/test-alloc-extra/scanner.l flex-2.5.34-pm/tests/test-alloc-extra/scanner.l
--- flex-2.5.34/tests/test-alloc-extra/scanner.l	2007-09-10 08:17:45.000000000 +0200
+++ flex-2.5.34-pm/tests/test-alloc-extra/scanner.l	2008-02-25 18:09:14.000000000 +0100
@@ -44,7 +44,7 @@ struct Check {
 static void check_extra ( yyscan_t  scanner );
 
 /* Special yyalloc */
-void *yyalloc ( size_t size, yyscan_t  scanner );
+void *yyalloc ( yy_size_t size, yyscan_t  scanner );
 
 %}
 
@@ -60,7 +60,7 @@
     return 0;
 }
 
-void *yyalloc(size_t size, yyscan_t scanner)
+void *yyalloc(yy_size_t size, yyscan_t scanner)
 {
     struct Check *check;
     check = testget_extra(scanner);
diff -urp flex-2.5.34/tests/test-c++-multiple-scanners/scanner-1.l flex-2.5.34-pm/tests/test-c++-multiple-scanners/scanner-1.l
--- flex-2.5.34/tests/test-c++-multiple-scanners/scanner-1.l	2002-10-01 17:10:12.000000000 +0200
+++ flex-2.5.34-pm/tests/test-c++-multiple-scanners/scanner-1.l	2008-02-25 16:54:48.000000000 +0100
@@ -1,23 +1,25 @@
- //  This file is part of flex.
- //  
- //  Redistribution and use in source and binary forms, with or without
- //  modification, are permitted provided that the following conditions
- //  are met:
- //  
- //  1. Redistributions of source code must retain the above copyright
- //     notice, this list of conditions and the following disclaimer.
- //  2. Redistributions in binary form must reproduce the above copyright
- //     notice, this list of conditions and the following disclaimer in the
- //     documentation and/or other materials provided with the distribution.
- //  
- //  Neither the name of the University nor the names of its contributors
- //  may be used to endorse or promote products derived from this software
- //  without specific prior written permission.
- //  
- //  THIS SOFTWARE IS PROVIDED ``AS IS'' AND WITHOUT ANY EXPRESS OR
- //  IMPLIED WARRANTIES, INCLUDING, WITHOUT LIMITATION, THE IMPLIED
- //  WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
- //  PURPOSE.
+/*
+ *  This file is part of flex.
+ *  
+ *  Redistribution and use in source and binary forms, with or without
+ *  modification, are permitted provided that the following conditions
+ *  are met:
+ *  
+ *  1. Redistributions of source code must retain the above copyright
+ *     notice, this list of conditions and the following disclaimer.
+ *  2. Redistributions in binary form must reproduce the above copyright
+ *     notice, this list of conditions and the following disclaimer in the
+ *     documentation and/or other materials provided with the distribution.
+ *  
+ *  Neither the name of the University nor the names of its contributors
+ *  may be used to endorse or promote products derived from this software
+ *  without specific prior written permission.
+ *  
+ *  THIS SOFTWARE IS PROVIDED ``AS IS'' AND WITHOUT ANY EXPRESS OR
+ *  IMPLIED WARRANTIES, INCLUDING, WITHOUT LIMITATION, THE IMPLIED
+ *  WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+ *  PURPOSE.
+ */
 
 %{
 #include "config.h"
diff -urp flex-2.5.34/tests/test-c++-multiple-scanners/scanner-2.l flex-2.5.34-pm/tests/test-c++-multiple-scanners/scanner-2.l
--- flex-2.5.34/tests/test-c++-multiple-scanners/scanner-2.l	2006-10-23 00:17:38.000000000 +0200
+++ flex-2.5.34-pm/tests/test-c++-multiple-scanners/scanner-2.l	2008-02-25 16:55:00.000000000 +0100
@@ -1,23 +1,25 @@
- //  This file is part of flex.
- //  
- //  Redistribution and use in source and binary forms, with or without
- //  modification, are permitted provided that the following conditions
- //  are met:
- //  
- //  1. Redistributions of source code must retain the above copyright
- //     notice, this list of conditions and the following disclaimer.
- //  2. Redistributions in binary form must reproduce the above copyright
- //     notice, this list of conditions and the following disclaimer in the
- //     documentation and/or other materials provided with the distribution.
- //  
- //  Neither the name of the University nor the names of its contributors
- //  may be used to endorse or promote products derived from this software
- //  without specific prior written permission.
- //  
- //  THIS SOFTWARE IS PROVIDED ``AS IS'' AND WITHOUT ANY EXPRESS OR
- //  IMPLIED WARRANTIES, INCLUDING, WITHOUT LIMITATION, THE IMPLIED
- //  WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
- //  PURPOSE.
+/*
+ *  This file is part of flex.
+ *  
+ *  Redistribution and use in source and binary forms, with or without
+ *  modification, are permitted provided that the following conditions
+ *  are met:
+ *  
+ *  1. Redistributions of source code must retain the above copyright
+ *     notice, this list of conditions and the following disclaimer.
+ *  2. Redistributions in binary form must reproduce the above copyright
+ *     notice, this list of conditions and the following disclaimer in the
+ *     documentation and/or other materials provided with the distribution.
+ *  
+ *  Neither the name of the University nor the names of its contributors
+ *  may be used to endorse or promote products derived from this software
+ *  without specific prior written permission.
+ *  
+ *  THIS SOFTWARE IS PROVIDED ``AS IS'' AND WITHOUT ANY EXPRESS OR
+ *  IMPLIED WARRANTIES, INCLUDING, WITHOUT LIMITATION, THE IMPLIED
+ *  WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+ *  PURPOSE.
+ */
 
 %{
 #include "config.h"
diff -urp flex-2.5.34/tests/test-pthread/scanner.l flex-2.5.34-pm/tests/test-pthread/scanner.l
--- flex-2.5.34/tests/test-pthread/scanner.l	2007-05-12 17:27:33.000000000 +0200
+++ flex-2.5.34-pm/tests/test-pthread/scanner.l	2008-02-25 17:02:03.000000000 +0100
@@ -52,7 +52,7 @@ static int process_text(char* s, yyscan_
 %option reentrant
 %option warn
 
-    /* Arbitrary states.*/
+/* Arbitrary states.*/
 %x STATE_1
 %x STATE_2
 
