Only in flex-2.5.33-pm: flex-2.5.33
diff -urp flex-2.5.33/Makefile.in flex-2.5.33-pm/Makefile.in
--- flex-2.5.33/Makefile.in	2006-02-21 03:45:12.000000000 +0100
+++ flex-2.5.33-pm/Makefile.in	2007-01-31 15:06:32.000000000 +0100
@@ -292,6 +292,7 @@ SUBDIRS = \
 
 localedir = $(datadir)/locale
 AM_CPPFLAGS = -DLOCALEDIR=\"$(localedir)\" -I@includedir@ -I$(top_srcdir)/intl
+AM_CFLAGS = -fPIC
 
 # Run GNU indent on sources. Don't run this unless all the sources compile cleanly.
 #
Only in flex-2.5.33-pm: Makefile.in~
