--- flex-2.5.33/gen.c
+++ flex-2.5.33/gen.c
@@ -1897,7 +1897,7 @@ void make_tables ()
 			outn ("\t\t{ \\");
 			outn ("\t\tint c = '*'; \\");
 			outn ("\t\tsize_t n; \\");
-			outn ("\t\tfor ( n = 0; n < max_size && \\");
+			outn ("\t\tfor ( n = 0; n < (size_t) max_size && \\");
 			outn ("\t\t\t     (c = getc( yyin )) != EOF && c != '\\n'; ++n ) \\");
 			outn ("\t\t\tbuf[n] = (char) c; \\");
 			outn ("\t\tif ( c == '\\n' ) \\");

