diff -urp flex-2.5.33/Makefile.in flex-2.5.33-pm/Makefile.in
--- flex-2.5.33/Makefile.in	2007-05-30 10:57:18.000000000 +0200
+++ flex-2.5.33-pm/Makefile.in	2007-05-30 10:57:11.000000000 +0200
@@ -291,7 +291,7 @@ SUBDIRS = \
 
 
 localedir = $(datadir)/locale
-AM_CPPFLAGS = -DLOCALEDIR=\"$(localedir)\" -I@includedir@ -I$(top_srcdir)/intl
+AM_CPPFLAGS = -DLOCALEDIR=\"$(localedir)\" -I$(top_srcdir)/intl
 AM_CFLAGS = -fPIC
 
 # Run GNU indent on sources. Don't run this unless all the sources compile cleanly.
